Makefile

```diff
@@ -341,6 +341,24 @@ setup-bigquery-vault:
 	make create-bigquery-emulator-tables
 	make enable-vault-transit
 
+IMAGES := gcr.io/distroless/base docker.io/arigaio/atlas:latest
+pull-images:
+	@echo "Checking and pulling images..."
+	@for image in $(IMAGES); do \
+		if docker image inspect $$image >/dev/null 2>&1; then \
+			echo "Image $$image already exists. Skipping..."; \
+		else \
+			echo "Pulling $$image"; \
+			docker pull $$image; \
+		fi; \
+	done
+
+force-pull-images:
```

Comment:
This command is used to force update images when developers have network connection.